### ISSUE TITLE
Cluster | The UI is showing the package validation % only after reload

### ISSUE THREAD

**Issue by liranmauda** [noobaa/noobaa-core#5089]
### Environment info
- Version: **2.9.0-aa129f6**
- Deployment: **AZURE**
- Customer: **QA**

### Actual behavior
1. After successful upload on upgrade package, the UI is showing the validation % only after reload 
- we are trying to upgrade from 2.9.0-aa129f6 to itself.

### Expected behavior
1. validation %  should start automatically.

### Screenshots or Logs or other output that would be helpful

![no_validation_summery](https://user-images.githubusercontent.com/28217223/44331056-36a76580-a471-11e8-9fc9-a18bfa996209.gif)

![vlaidation_starts_after_reload](https://user-images.githubusercontent.com/28217223/44331196-8f76fe00-a471-11e8-97a3-a0b3af48d252.gif)

**Comment by nimrod-becker** [comment]
No longer relevant due to OVA deployment support removal. Upgrade will be done via k8s (either initiated by the user or not, it will still be an unmanaged path)
